src/lib/agent/PolykeyAgent.ts

```diff
@@ -231,12 +271,42 @@ class PolykeyAgent {
           case AgentMessageType.UPDATE_SECRET:
             response = await this.updateSecret(nodePath, subMessage);
             break;
+          case AgentMessageType.ADD_PEER:
+            response = await this.addPeer(nodePath, subMessage);
+            break;
+          case AgentMessageType.GET_PEER_INFO:
+            response = await this.getPeerInfo(nodePath, subMessage);
+            break;
+          case AgentMessageType.PING_PEER:
+            response = await this.pingPeer(nodePath, subMessage);
+            break;
+          case AgentMessageType.FIND_PEER:
+            response = await this.findPeer(nodePath, subMessage);
+            break;
+          case AgentMessageType.FIND_SOCIAL_PEER:
+            response = await this.findSocialPeer(nodePath, subMessage);
+            break;
+          case AgentMessageType.LIST_PEERS:
+            response = await this.listPeers(nodePath, subMessage);
+            break;
+          case AgentMessageType.TOGGLE_STEALTH:
+            response = await this.toggleStealth(nodePath, subMessage);
+            break;
+          case AgentMessageType.UPDATE_PEER_INFO:
+            response = await this.updatePeerInfo(nodePath, subMessage);
+            break;
+          case AgentMessageType.REQUEST_RELAY:
+            response = await this.requestRelay(nodePath, subMessage);
```

Comment:
All these methods should be broken up into different models. Maybe prototype this so to see if it makes it more cohesive. Right now too many different methods on Agent. Unless we expect this to be the router to subcommands... then that's a different story.

Reply:
yeah I agree there are too many methods right now in the agent class and many should have their own models and separate folders. Eventually when we get the agent control happening via gRPC, a lot of this code with go away.

Reply:
https://github.com/MatrixAI/js-polykey/issues/102